specification/_json_spec/inference.post.eis_chat_completion.json

```diff
@@ -0,0 +1,35 @@
+{
+  "inference.post.eis_chat_completion": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html",
+      "description": "Perform a chat completion task via the Elastic Inference Service (EIS)"
+    },
+    "stability": "stable",
+    "visibility": "public",
+    "headers": {
+      "accept": ["application/json"],
+      "content_type": ["application/json"]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_inference/chat_completion/{eis_inference_id}/_stream",
+          "methods": ["POST"],
+          "parts": {
+            "task_type": {
```

Comment:
I think we can remove the `task_type` object since we've defined `chat_completion` in the URL.